packages/components/src/tag/tag.tsx

```diff
@@ -38,7 +38,13 @@ const Tag = (props: Props, ref: Ref<HTMLButtonElement>) => {
       disabled={disabled}
       ref={ref}
       data-selected={selected}
-      className={styles({ size, selected, disabled, iconOnly })}
+      className={styles({
+        size,
+        selected,
+        disabled,
+        iconOnly,
+        pressable: Boolean(onClick),
```

Comment:
i don't think the fix is that easy, if element is not pressable then it shouldn't be a button (which is still focusable even though cursor is default)

Reply:
So, I can see two options, either change it to `div` or make button `disabled`?